src/Docker.PowerShell/Cmdlets/StartContainer.cs

```diff
@@ -33,9 +34,9 @@ public class StartContainer : MultiContainerOperationCmdlet
                     throw new ApplicationFailedException("The container has already started.");
                 }
 
-                if (PassThru.ToBool())
+                if (PassThru)
                 {
-                    WriteObject(await ContainerOperations.GetContainerByIdOrName(id, DkrClient));
+                    WriteObject((await ContainerOperations.GetContainersByIdOrName(id, DkrClient)).Single());
```

Comment:
`.Single()` will be a terrible error if its a wrong id/name. Don't we want to handle this slightly better?